.github/workflows/pull_request_build.yml

```diff
@@ -29,10 +29,12 @@ jobs:
         id: cache-data
         uses: actions/cache@v2
         with:
-          path: data/cats_dogs
-          key: ${{ hashFiles('data/cats_dogs.dvc') }}
+          path: .dvc/cache
+          key: ${{ runner.os }}-data-${{ hashFiles('**/*.dvc') }}
```

Comment:
I would put here an if checking the system - now, on windows we will get additional work done